Cleanup README (#87)
RedHatQE/openshift-virtualization-tests: README.md

# openshift-virtualization-tests

> This repository contains tests to verify the functionality of OpenShift with CNV installation.

Author: Yeah, agree, done

## virtctl

> `virtctl` binary is to be downloaded from `consoleCliDownloads` resource of the cluster under test.

Reviewer: ```suggestion
`virtctl` binary should be downloaded from `consoleCliDownloads` resource of the cluster under test.
```

Author: Done

## oc

> `oc` client is to be downloaded from `consoleCliDownloads` resource of the cluster under test.

Reviewer: ```suggestion
`oc` client should be downloaded from `consoleCliDownloads` resource of the cluster under test.
```

Author: Done

To run the chaos tests the following command needs to be run:

```bash

> make tests PYTEST_ARGS="-m chaos"

Author: Is this `make tests PYTEST_ARGS` still relevant at all?

Author: Updated
